what-the-stack/build.gradle

```diff
@@ -54,6 +67,10 @@ dependencies {
     implementation libs.startup
     implementation libs.insetter
```

Comment:
Perhaps the dependency on `insetter` can be removed, since we are not using it anymore.

Reply:
Make sure to remove it from the project level `build.gradle` file too.

Reply:
@haroldadmin Do you want me to delete `activity_what_the_stack.xml` too?